src/control/common/env_utils.go

```diff
@@ -63,6 +64,7 @@ func ScrubProxyVariables() {
 	proxyVars := []string{
 		"http_proxy", "https_proxy", "ftp_proxy", "socks_proxy", "no_proxy",
 		"HTTP_PROXY", "HTTPS_PROXY", "FTP_PROXY", "SOCKS_PROXY", "NO_PROXY",
+		"DAOS_HTTPS_PROXY",
```

Comment:
I'm not familiar enough, but do we really need this change in the control plane? Since we ultimately
```
export HTTPS_PROXY=DAOS_HTTPS_PROXY
```

Reply:
Yeah, I don't think we want or need this here, as it's specific to the HPE CI environment. The utility of this helper is for allowing e.g. the dmg utility to invoke gRPC methods on an internal server without accidentally going through the proxy.

Reply:
https://github.com/daos-stack/daos/pull/16496/commits/6875c09a06236f8af6ac6a8d06e699f1ac3e12d8